modules/29_CES_parameters/calibrate/preloop.gms

```diff
@@ -816,20 +547,20 @@ loop (cesOut2cesIn("inco",in)$( NOT sameas(in,"lab") ),
 
       if (sm_tmp gt 1,
         put "  ", t.tl, " ", regi.tl, in.tl:>4, "   ",
-	    pm_cesdata(t,regi,in,"quantity"), " x ",
-	    pm_cesdata(t,regi,in,"price"), " > ",
-	    pm_cesdata(t,regi,"inco","quantity"), " -> ";
+         pm_cesdata(t,regi,in,"quantity"), " x ",
+         pm_cesdata(t,regi,in,"price"), " > ",
+         pm_cesdata(t,regi,"inco","quantity"), " -> ";
 
         pm_cesdata(t,regi,in,"price")
-	= ( pm_cesdata(t,regi,"inco","quantity")
-	  - sum(cesOut2cesIn2("inco",in2)$( NOT sameas(in,in2) ),
-	      pm_cesdata(t,regi,in2,"quantity")
-	    * pm_cesdata(t,regi,in2,"price")
-	    )
-	  )
-	/ pm_cesdata(t,regi,in,"quantity");
-
-	put pm_cesdata(t,regi,in,"price") /;
+    = ( pm_cesdata(t,regi,"inco","quantity")
+      - sum(cesOut2cesIn2("inco",in2)$( NOT sameas(in,in2) ),
+          pm_cesdata(t,regi,in2,"quantity")
+        * pm_cesdata(t,regi,in2,"price")
+        )
+      )
+    / pm_cesdata(t,regi,in,"quantity");
+
+    put pm_cesdata(t,regi,in,"price") /;
```

Comment:
Fix indentation.

Reply:
> * Use indentations to make the structure more readable

[[↑]](https://github.com/remindmodel/remind/blob/5c197bd42b7f4fc66f1bc9073896f632ea656f8a/main.gms#L144)

Why is the assignment not indented with the thing being assigned to?